components/wso2is.key.manager.operations.endpoint/src/main/java/org/wso2/is/key/manager/operations/endpoint/dcr/service/DCRMService.java

```diff
@@ -162,6 +170,34 @@ public ExtendedApplication updateApplication(ExtendedApplicationUpdateRequest up
         return buildResponse(getApplicationById(clientId));
     }
 
+    /**
+     * Update service provider property list
+     *
+     * @param sp                     Service provider
+     * @param applicationDisplayName Display name of the application
+     */
+    public void updateServiceProviderPropertyList(ServiceProvider sp, String applicationDisplayName) {
```

Comment:
make this private if not used from other classes

Reply:
Fixed with https://github.com/wso2-extensions/apim-km-wso2is/pull/67